docs/pager/config.md

```diff
@@ -23,6 +23,7 @@ The list of existing options are:
 | filterFieldWhitelist       | array          |               | FiltrationSubscriber                            |
 | filterFieldParameterName   | string         | filterParam   | FiltrationSubscriber                            |
 | filterValueParameterName   | string         | filterValue   | FiltrationSubscriber                            |
+| pageOutOfRange             | string         | ignore        | PaginationSubscriber                            |
```

Comment:
I am not sure about list of subscribers.

Reply:
What you're not sure about?

Reply:
Whether I am not missing anything.

Reply:
Well, I have to admin that last column is not super-clear to me. For example, I don't get why `pageParameterName` should be for SortableSubscriber only...
I think we can go with PaginationSubscriber